src/torrent.cpp

```diff
@@ -8244,16 +8244,16 @@ namespace libtorrent
 			m_seeding_time += lost_seconds;
 		}
 		m_became_seed = clamped_subtract_u16(m_became_seed, seconds);
-
-		if (m_finished_time < seconds && is_finished())
+		if (m_finished_time < seconds && is_finished() && !is_paused())
```

Comment:
adding ``&& !is_paused()`` here changes the semantics of finished_time, doesn't it? what's the rationale for this change?